docs/registry/attributes/workflow.md

```diff
@@ -0,0 +1,104 @@
+| <a id="workflow-task-version" href="#workflow-task-version">`workflow.task.version`</a> | string | Version of the task | `2.1` | ![Development](https://img.shields.io/badge/-development-blue) |
+| <a id="workflow-trigger-id" href="#workflow-trigger-id">`workflow.trigger.id`</a> | string | Unique ID for this trigger which can be used to correlate trigger usage | `5c6e9a4b-69ab-499f-b4a7-4cf5c8720d66` | ![Development](https://img.shields.io/badge/-development-blue) |
+| <a id="workflow-trigger-name" href="#workflow-trigger-name">`workflow.trigger.name`</a> | string | Name of the trigger | `Nightly build` | ![Development](https://img.shields.io/badge/-development-blue) |
+| <a id="workflow-trigger-schedule" href="#workflow-trigger-schedule">`workflow.trigger.schedule`</a> | string | A string containing the schedule period as [Cron Expression](https://docs.oracle.com/cd/E12058_01/doc/doc.1014/e12030/cron_expressions.htm). | `0/5 * * * ? *` | ![Development](https://img.shields.io/badge/-development-blue) |
+| <a id="workflow-trigger-type" href="#workflow-trigger-type">`workflow.trigger.type`</a> | string | Type of trigger that was called | `cron` | ![Development](https://img.shields.io/badge/-development-blue) |
+| <a id="workflow-type" href="#workflow-type">`workflow.type`</a> | string | Type of the workflow | `DSL` | ![Development](https://img.shields.io/badge/-development-blue) |
+| <a id="workflow-url-full" href="#workflow-url-full">`workflow.url.full`</a> | string | The [URL](https://wikipedia.org/wiki/URL) of the workflow, providing the complete address in order to locate and identify the workflow. | `https://github.com/open-telemetry/semantic-conventions/actions/runs/9753949763?pr=1075` | ![Development](https://img.shields.io/badge/-development-blue) |
+| <a id="workflow-version" href="#workflow-version">`workflow.version`</a> | string | Version of the workflow | `2.1` | ![Development](https://img.shields.io/badge/-development-blue) |
+
+**[1] `workflow.task.annotation.<key>`:** The `<key>` being the annotation name, the value being the annotation value, even if the value is empty.
+
+---
+
+`workflow.execution.result` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value  | Description | Stability |
+|---|---|---|
+| `cancellation` | The task run was cancelled, eg. by a user manually cancelling the task run. | ![Development](https://img.shields.io/badge/-development-blue) |
+| `error` | The task run failed due to an error in the CICD system, eg. due to the worker being killed. | ![Development](https://img.shields.io/badge/-development-blue) |
+| `failure` | The task run did not finish successfully, eg. due to a compile error or a failing test. Such failures are usually detected by non-zero exit codes of the tools executed in the task run. | ![Development](https://img.shields.io/badge/-development-blue) |
```

Comment:
There should only  be one out of `error` and `failure` Also all of the descriptions appear to be copy and pasted from CICD.